src/main/java/uk/gov/justice/hmpps/casenotes/alertbackfill/CaseNoteAlertResponse.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.casenotes.alertbackfill
+
+import java.time.LocalDate
+import java.time.LocalDate.now
+import java.time.LocalDateTime
+
+data class CaseNoteAlertResponse(val content: List<CaseNoteAlert>)
+
+data class CaseNoteAlert(
+  val type: CodedDescription,
+  val subType: CodedDescription,
+  val prisonCode: String?,
+  val activeFrom: LocalDate,
+  val activeTo: LocalDate?,
+  val createdAt: LocalDateTime,
+  val madeInactiveAt: LocalDateTime?,
+) {
+  fun isActive(): Boolean = activeTo?.isAfter(now()) ?: true
+  fun madeInactive(): Boolean = activeTo != null && madeInactiveAt != null
```

Comment:
Is the activeTo != null check needed here? If it has a madeInactiveAt, it will be inactive and have an activeTo. They can't be reactivated